focal/4_gs1_identification_examples.html

```diff
@@ -0,0 +1,94 @@
+<section>
+      <h3>Focal Use Case: Chain of GS1 Credentials to Identify a Trade Item</h3>
+
+      <p>
+            These examples are based on the document <a href="https://ref.gs1.org/gs1/vc/data-model/" rel="nofollow">"GS1 Verifiable Credentials - Data Model and Validations"</a>. The examples use Verifiable Credentials Data Model 2.0.
+      </p>
+      <p>
+            The key artefact is the last one; it declares the existence of a GTIN, around which other Verifiable Credentials may be issued to declare data about the trade item (brand, size and unit of measure, ingredients, dimensions and weights, etc.). Validation of the artefact requires validating all the credentials that come before it, identified in each case by "extendsCredential".
+      </p>
+      
+      <pre class="example" title="GS1 Prefix 952 Licensed by GS1 Global Office to GS1 Utopia">
+{
+      "@context": [
+            "https://www.w3.org/2018/credentials/v1",
```

Comment:
Should be updated to the v2 context. https://www.w3.org/ns/credentials/v2

Reply:
Fixed.